modules/cudafilters/src/filtering.cpp

```diff
@@ -69,6 +69,8 @@ Ptr<Filter> cv::cuda::createBoxMinFilter(int, Size, Point, int, Scalar) { throw_
 Ptr<Filter> cv::cuda::createRowSumFilter(int, int, int, int, int, Scalar) { throw_no_cuda(); return Ptr<Filter>(); }
 Ptr<Filter> cv::cuda::createColumnSumFilter(int, int, int, int, int, Scalar) { throw_no_cuda(); return Ptr<Filter>(); }
 
+Ptr<Filter> cv::cuda::createMedianFiltering(InputArray, OutputArray, int,int) { throw_no_cuda(); return Ptr<Filter>();}
```

Comment:
Done.


________________________________
From: Vladislav Vinogradov <notifications@github.com>
Sent: Monday, January 25, 2016 3:59 AM
To: Itseez/opencv
Cc: Green, Oded
Subject: Re: [opencv] Implemented median filter in CUDA (#5888)


In modules/cudafilters/src/filtering.cpp<https://github.com/Itseez/opencv/pull/5888#discussion_r50667338>:

> @@ -69,6 +69,8 @@ Ptr<Filter> cv::cuda::createBoxMinFilter(int, Size, Point, int, Scalar) { throw_
>  Ptr<Filter> cv::cuda::createRowSumFilter(int, int, int, int, int, Scalar) { throw_no_cuda(); return Ptr<Filter>(); }
>  Ptr<Filter> cv::cuda::createColumnSumFilter(int, int, int, int, int, Scalar) { throw_no_cuda(); return Ptr<Filter>(); }
>
> +Ptr<Filter> cv::cuda::createMedianFiltering(InputArray, OutputArray, int,int) { throw_no_cuda(); return Ptr<Filter>();}


createMedianFilter

-
Reply to this email directly or view it on GitHub<https://github.com/Itseez/opencv/pull/5888/files#r50667338>.